com.unity.render-pipelines.universal/Runtime/Passes/GBufferPass.cs

```diff
@@ -68,10 +70,11 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
                 RenderTextureDescriptor gbufferSlice = cameraTextureDescriptor;
                 gbufferSlice.depthBufferBits = 0; // make sure no depth surface is actually created
                 gbufferSlice.stencilFormat = GraphicsFormat.None;
-                gbufferSlice.graphicsFormat = m_DeferredLights.GetGBufferFormat(i);
+                gbufferSlice.graphicsFormat = gbufferFormats[i];
                 cmd.GetTemporaryRT(m_DeferredLights.GbufferAttachments[i].id, gbufferSlice);
```

Comment:
I guess this is one example to justify my worry with the API design you have today.

This entire piece of code will be replaced by RTHandle in the future. So then gbufferSlice.graphicsFormat will not exist. If you have two sets of API taking per-attachement data (ConfigureDescriptor(formats), ConfigureTarget(rtIDs)) then you have to loop the RTHandle array just to get those arrays. 

So, in essence making your API today pass the GraphicsFormat array in ConfigureTarget will make it easy to switch to ConfigureTarget taking an RTHandle array once we support it.